### DISCUSSION TITLE
New discussion forum rules

### DISCUSSION THREAD

**Discussion by fmw42** [ImageMagick/ImageMagick#1878] [General]
Please clarify what topics are acceptable and un-acceptable. Are there any other guidelines or rules for topics?

**Comment by dlemstra** [comment]
Not sure what you are asking. We allow the same kind of questions as we do on our forum. The big difference is that bugs should be reported as an [issue ](https://github.com/ImageMagick/ImageMagick/issues) here instead.

**Comment by fmw42** [comment]
That is still pretty much wide open. I just wanted to be sure users understand what limitations, if any, there might be.

**Comment by dlemstra** [comment]
I am not aware of any limitations in the current situation. I think we should mention it when the situation occurs?
